pallets/ethereum-events/src/lib.rs

```diff
@@ -1707,4 +1686,63 @@ pub mod migrations {
         log::info!("ℹ️  Migrated Ethereum event's NFT contract addresses successfully");
         return consumed_weight
     }
+
+    pub fn migrate_to_bridge_contract<T: Config>() -> frame_support::weights::Weight {
+        sp_runtime::runtime_logger::RuntimeLogger::init();
+        log::info!("ℹ️  Ethereum events pallet data migration invoked");
+
+        let consumed_weight = Weight::from_ref_time(0);
+
+        let lift_address = LiftingContractAddress::<T>::get();
+        let validator_manager_address = ValidatorManagerContractAddress::<T>::get();
+        let pub_root_address = PublishRootContract::<T>::get();
+
+        if !lift_address.is_zero() {
+            log::info!("ℹ️  Checking if zero");
+            if AvnBridgeContractAddress::<T>::get().is_zero() {
+                <AvnBridgeContractAddress<T>>::put(lift_address);
+                log::info!("ℹ️  Updated bridge contract address successfully");
+                StorageVersion::<T>::put(Releases::V4_0_0);
+                consumed_weight.saturating_add(T::DbWeight::get().writes(1));
```

Comment:
```suggestion
                consumed_weight.saturating_add(T::DbWeight::get().read_writes(1, 2));
```
Each of these blocks do one read and two writes.